Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM jred/nodejs:12.17 as builder
+LABEL maintainer="jared.a.fowler@gmail.com"
+LABEL description="Nodejs docker base image"
+
+
+FROM node:12.17-buster-slim
+LABEL maintainer.fola="darragh@entrotech.net"
```

Comment:
you might consider creating a repo on docker hub: [hub.docker.com](hub.docker.com)

where we can build push and pull hackforla food-oasis docker images. Not required at this point.

Reply:
username: foodoasis
email: foodoasis+hub@hackforla.org

Reply:
I can give you the pw if needed.